src/rviz/failed_display.cpp

```diff
@@ -70,6 +70,11 @@ void FailedDisplay::load( const Config& config )
 }
 
 void FailedDisplay::save( Config config )
+{
+  save(config);
```

Comment:
Shouldn't this use a `const_cast` for `this` to actually redirect to the `const`-version and avoid an infinite loop?